### DISCUSSION TITLE
Is there a release that contains the following PR?

### DISCUSSION THREAD

**Discussion by ataghavi** [ent/ent#3229] [Q&A] [answered]
https://github.com/ent/ent/pull/2849

We need to pull in this fix, but it seems like the most recent release does not contain this fix.

**Comment by a8m** [comment, marked as answer]
Hey @ataghavi! The master branch was tagged a few days ago. You can pull the `latest` version or `v0.11.5`.

**Comment by ataghavi** [comment]
Awesome, thank you. I do have a question about the behavior for that PR, if this is an ok place to ask it. 

I am using ent for a mysql integration, and we have a use case where we want to do a CreateBulk and basically overwrite the rows that are in the db if there is a conflict. However, we are running into an issue where the new value for a field is "nil", then that field is keeping the value in the db rather than being overwritten with nil. 

And this is actually only happening for a string value, and not for a decimal value (the decimal value does get overwritten with nil). Thoughts on why this might be happening and how to fix it?

**Comment by ataghavi** [comment]
Also, if I do the following:

client.CountingSessionLine.CreateBulk(lineBuilders...).OnConflict().ClearOverriddenBy().UpdateNewValues().Exec(ctx)

Then everything works fine (OverriddenBy is the field that was not getting nilled out when it needed to be). My understanding is that "OnConflict().ClearOverriddenBy()" will clear the existing overriddenBy field when there is a conflict, and then write the new value for overriddenBy for the update. Is that correct?
